mount/mount_darwin.go

```diff
@@ -0,0 +1,59 @@
+//go:build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// Mount to the provided target.
+func (m *Mount) mount(target string) error {
+	var commandName string
+	if m.Type == "bind" {
+		// macOS doesn't natively support bindfs/nullfs
+		// The way to emulate it is via FUSE fs named "bindfs"
+		commandName = "bindfs"
```

Comment:
Wait I might've missed this first time around, do we use commandName after this?

Reply:
Well rather, we always supply a hardcoded "bindfs" for the actual exec.Command, but I see we use commandName in the error msg. We should replace the command also with it

Reply:
Whoopsie-doopsie :D Fixed